core/server/middleware/error-handler.js

```diff
@@ -81,6 +82,8 @@ _private.HTMLErrorRenderer = function HTMLErrorRender(err, req, res, /*jshint un
     // This ensures that no matter what res.render will work here
     if (_.isEmpty(req.app.engines)) {
         req.app.engine('hbs', _private.createHbsEngine());
+        req.app.set('view engine', 'hbs');
+        req.app.set('views', config.get('paths').defaultViews);
```

Comment:
We agreed on keeping it like it is, because there are other cases where we want to show `Resource could not be found`. 

Here is how i tested this use case:
- I've removed the `admin/views` html files and restarted Ghost
- Then i have added the inheritance message property to the html error handler to get the underlying error message